docs/gallery-submission.md

```diff
@@ -140,82 +132,129 @@ To make things easy, we created templates for your custom installation instructi
 
----
-## **Go to Market**
+#### Scheduled publish date
 
-### Step 7: Prepare Go-to-Market activities and get content approved by Tableau
+If you would like your connector to be published on a certain date, select that date. Be aware when selecting a date that the approval process for a connector takes two to six weeks.
 
-Many partners choose to put out blogs and a press release once they offer custom connectors for their customers in Tableau Exchange. To fully promote your work, we recommend collecting a public customer case study that highlights the advantages and usefulness of your new connector. Not only does Tableau have mechanisms and channels to promote your customer case study, but historical data shows that referencing customer case studies generate on average five times more traffic than standalone blogs.
+### Step 5: Submit source code and test artifacts via GitHub
 
-We are happy to support partners looking to issue Tableau-specific press releases. We require that any partner press release mentioning Tableau is first submitted for approval to ensure that the Tableau positioning and branding are accurate.
+#### GitHub username or email
 
-**Tableau PR requires a 2-week approval timeline for press releases.**
+Share your GitHub username or email with your TPN account manager, and we will create a GitHub repository for you to share your source code and test artifacts. 
 
-The process:
+#### Connector source code
 
-1. Partner drafts a press release using the Tableau Partner Brand Guidelines (available to partners in the PDC).
+Provide the unzipped source files for your connector so we can review each file.
 
-Note: The release should not include the Tableau corporate boilerplate.
-2. Partner submits the draft to the Partner Marketing team via email (partnermarketing@tableau.com) for initial review.
-3. The Partner Marketing team contacts Tableau PR for a full review and legal sign-off.
+#### TDVT files 
 
-**Note** : We consider quote attribution from a Tableau spokesperson on a case-by-case basis. Press releases that announce milestone product launches and significant customer successes are typically reviewed to determine relevant quote attribution. For these types of releases, the partner can include a proposed Tableau quote as part of the draft for consideration.
+[The TDVT suite]({{ site.baseurl }}/docs/tdvt) produces a number of important files required for a review. Provide the following files.
 
+- TDVT results csv file (test_results_combined.csv)
+- TDVT logs (tdvt_log_combined.txt, tabquery_logs.zip, and tdvt_actuals_combined.zip)
+- TDVT environment (INI config file)
 
----
-## **Beta Process**
+TDVT tests can be skipped for extract-only connectors.
 
-### Step 8: Release connector in Tableau Exchange for beta testing.
+In most cases, TDVT should be passing more than 90% of test cases to be accepted.
```

Comment:
Can you also add this beside 90% of test cases to be accepted - All tests marked as required here - https://github.com/tableau/connector-plugin-sdk/blob/master/tdvt/tdvt/metadata/priority.csv
 should pass or acknowledged as not supported or known bug for the underlying data source
@pvanderknyff - Is there a better link to show which tests are required ?